deployments/monitoring-env-devops.nix

```diff
@@ -1,11 +1,42 @@
-{
+{ IOHKaccessKeyId, ... }:
+let
+  iohklib = import ../lib.nix;
+  mkUplink = iohklib.mkMkUplink {
+    central = "192.168.20.1";
+    subnet = "192.168.20";
+    # TODO, `monitoring-ip` will be wrong if monitoring isnt using an elastic ip by that name
+    endpoint = "monitoring-ip:51820";
+  };
+in {
   require = [ ./monitoring.nix ];
-  monitoring = { ... }:
+  a1 = mkUplink 10 ../static/a1.wgprivate;
```

Comment:
this stuff needs to be undone and moved to infra for all packet.net servers.

Reply:
this was mainly a testing ground for the concept, and matching code has now been put into the infra deployments
i'm thinking it can be ok to leave wireguard in this devops deployment, for future testing?